res/database/Default/lang/pl.xml

```diff
@@ -28,30 +28,67 @@
 		<person guid="various-puppeteers" first_name="" last_name="różni animatorzy" />
 		<person guid="various-models" first_name="" last_name="różni modele" />
 		<person guid="various-musicians" first_name="" last_name="różni muzycy" />
+		<person guid="various-narrators" first_name="" last_name="różni narratorzy" />
+		<person guid="various-guests" first_name="" last_name="różni goście" />
 		<person guid="1994aa3a-23c3-48ce-b20a-d1f021df8c63" first_name="Królowa Svetlana" last_name="ze Szwecji" />
 		<person guid="2994ff3d-ffc4-f8ce-c25a-e1g0d1df8c64" first_name="bezrobotny mężczyzna o imieniu" last_name="&quot;Woozy&quot;" />
-		<person guid="f14d6ba0-0db5-4a76-a4d3-daba1603e4f6" first_name="Johnny" last_name="Jobless" />
+		<person guid="f14d6ba0-0db5-4a76-a4d3-daba1603e4f6" first_name="Bartek" last_name="Bezrobotny" />
 		<person guid="f43f2b92-43cd-4fc8-93aa-25adc647fb63" first_name="Elmer" last_name="Rednose" />
 
 		<person guid="51397467-86e2-4c5f-9226-d93ecf9288a6" first_name="Nikita Sergejevich" last_name="Chruszczow" />
 		<person guid="6dd0a9b9-96a0-4712-83f0-f6388bdc5bec" first_name="Michaił" last_name="Gorbaczow" />
+		<person guid="f1bc5e03-2dbc-4219-88a5-6fa1dab4cb79" first_name="Fëdot Mikhailovič" last_name="Dostoeshkev" />
+
+		<person guid="SpeedMinister-celebritypeople-KarlHeinzDorfmichel" first_name="John" last_name="Silly Billy" /> <!-- TODO: should probably be something traditionally Polish -->
 	</persons>
 	<roles>
 		<!--
 			<role guid="script-roles-ron-001" first_name="Vincent" last_name="Count" title="" />
 		-->
 
 		<role guid="a00ea58a-4a9f-466e-9fe5-cf8b53730b60" first_name="Królewna Śnieżka" last_name="" />
-		<role guid="9faa394a-3c7e-4943-9236-7ccf70db6c6e" first_name="Królewna Śnieżka" last_name="" />
-		<role guid="e38dc13c-ce6a-4a0f-98f5-fd5cec332bf6" first_name="Róża czerwona" last_name="" />
+		<role guid="9faa394a-3c7e-4943-9236-7ccf70db6c6e" first_name="Białośnieżka" last_name="" />
```

Comment:
Królewna Śnieżka is correct.

Reply:
This is "Królewna Śnieżka"
![image](https://github.com/user-attachments/assets/2aa7be16-882c-4791-a6e5-31d5443a67e0)

And this is "Białośnieżka":
![image](https://github.com/user-attachments/assets/d6f3e5c2-356b-4c43-a307-8a3924152f62)

In Germany we distinguish between "Schneewittchen" and "Schneeweißchen and Rosenrot". In English we don't

Nonetheless - at least for "Rose Red" it seems to be "`Różanka`" and not "`Róża czerwona`" - but this is removed anyways...

Reply:
And if it's about this position then yes you are right. 
Stay: Białośnieżka and Różanka.

Reply:
I have stuff like this "documented" in the comments in the `database_roles.xml" file (links to Wikipedia "Snow White" vs "Snow White and Rose Red") but it's probably too easy to miss, I should add XML comments to the English language file too I guess.

Generally, if you are unsure about how to translate something, take a look if there's some info or hint or explanation in the `comment`.